document_page_quality_manual/README.rst

```diff
@@ -1,3 +1,6 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
@elicoidal The template has https links as well:
https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst